libminifi/src/core/RecordField.cpp

```diff
@@ -0,0 +1,99 @@
+/**
+* Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
very minor, but a space is missing from the start of this line

Reply:
Updated in ddaff1ec9ab40fb6070d0274bba7bb4b9e226e13